keyboards/walletburner/cajal/cajal.c

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2020 Worldspawn <mcmancuso@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "cajal.h"
+
+void matrix_init_kb(void) {
+    setPinOutput(B5);
+    writePinLow(B5);
+    setPinOutput(B6);
+    writePinLow(B6);
+    setPinOutput(B7);
+    writePinLow(B7);
+    matrix_init_user();
+}
+
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    if(res) {
+        writePin(B5, led_state.caps_lock);
+        writePin(B6, led_state.num_lock);
+        writePin(B7, led_state.scroll_lock);
+    }
+    return res;
+}
+
```

Comment:
```suggestion
```

This can be replaced with the following in the `config.h` file for the board:
```c
#define LED_NUM_LOCK_PIN B6
#define LED_CAPS_LOCK_PIN B5
#define LED_SCROLL_LOCK_PIN B7
```